mkosi/__init__.py

```diff
@@ -5094,27 +5094,54 @@ def parse_ssh_agent(value: str) -> Optional[Path]:
         die(f"SSH agent socket {sock} is not an AF_UNIX socket")
     return sock
 
+USAGE = '''
```

Comment:
I know you prefer single quotes, but for triple-quoted strings [PEP 8](https://peps.python.org/pep-0008/#string-quotes) actually has a preference for them. ;)

Reply:
Done.